nestkernel/connection_manager.cpp

```diff
@@ -1432,9 +1435,13 @@ nest::ConnectionManager::set_have_connections_changed( const thread tid )
   // performance issues on supercomputers.
   if ( have_connections_changed_[ tid ].is_false() )
   {
-    std::string msg =
-      "New connections created, connection descriptors previously obtained using 'GetConnections' are now invalid.";
-    LOG( M_WARNING, "ConnectionManager", msg );
+    if ( has_get_connections_been_called_[ tid ].is_true() )
+    {
+      std::string msg =
+        "New connections created, connection descriptors previously obtained using 'GetConnections' are now invalid.";
+      LOG( M_WARNING, "ConnectionManager", msg );
+      has_get_connections_been_called_[ tid ].set_false();
```

Comment:
So we will only get a warning the first time `Connect()` is called after `GetConnections()`?

Reply:
Yes.